agent/engine/docker_image_manager_integ_test.go

```diff
@@ -667,19 +668,19 @@ func verifyImagesAreNotRemoved(imageManager *dockerImageManager, imageIDs ...str
 }
 
 func cleanupImagesHappy(imageManager *dockerImageManager) {
-	imageManager.client.RemoveContainer("test1", removeContainerTimeout)
-	imageManager.client.RemoveContainer("test2", removeContainerTimeout)
-	imageManager.client.RemoveContainer("test3", removeContainerTimeout)
-	imageManager.client.RemoveImage(test1Image1Name, imageRemovalTimeout)
-	imageManager.client.RemoveImage(test1Image2Name, imageRemovalTimeout)
-	imageManager.client.RemoveImage(test1Image3Name, imageRemovalTimeout)
+	imageManager.client.RemoveContainer(context.TODO(), "test1", removeContainerTimeout)
+	imageManager.client.RemoveContainer(context.TODO(), "test2", removeContainerTimeout)
+	imageManager.client.RemoveContainer(context.TODO(), "test3", removeContainerTimeout)
+	imageManager.client.RemoveImage(context.TODO(), test1Image1Name, imageRemovalTimeout)
+	imageManager.client.RemoveImage(context.TODO(), test1Image2Name, imageRemovalTimeout)
+	imageManager.client.RemoveImage(context.TODO(), test1Image3Name, imageRemovalTimeout)
 }
 
 func cleanupImagesThreshold(imageManager *dockerImageManager) {
-	imageManager.client.RemoveContainer("test1", removeContainerTimeout)
-	imageManager.client.RemoveContainer("test2", removeContainerTimeout)
-	imageManager.client.RemoveContainer("test3", removeContainerTimeout)
-	imageManager.client.RemoveImage(test2Image1Name, imageRemovalTimeout)
-	imageManager.client.RemoveImage(test2Image2Name, imageRemovalTimeout)
-	imageManager.client.RemoveImage(test2Image3Name, imageRemovalTimeout)
+	imageManager.client.RemoveContainer(context.TODO(), "test1", removeContainerTimeout)
```

Comment:
Just create context once as per my earlier comments and use it everywhere.

Reply:
OK